packages/las-sdk-core/src/client.spec.ts

```diff
@@ -116,12 +118,16 @@ describe('Documents', () => {
       const testContentType = 'image/jpeg';
       const testConsentId = consentId();
       const testBatchId = batchId();
+      const testDatasetId = datasetId();
       const createDocumentPromise = client.createDocument(testContent, testContentType, {
-        consentId: testConsentId,
         batchId: testBatchId,
+        consentId: testConsentId,
+        datasetId: testDatasetId,
```

Comment:
I was thinking about this earlier when dealing with the conflict of the UUID generator function names and wanting to do `const batchId = batchId()` instead of having to call the variables testBatchId etc. 

Should we rename the UUID functions to something like `generateBatchId` or `randomBatchId`?

Then you could also do
```
{
  batchId, // instead of batchId: testBatchId etc
  consentId,
  datasetId,
}
```

I was always annoyed by this personally, but somehow never did anything with it...